ThreatZone CDR & IOC Updates (#33470)
demisto/content: Packs/ThreatZone/ReleaseNotes/1_0_1.md

#### Integrations

##### ThreatZone

- Updated the Docker image to: *demisto/python3:3.10.13.89009*.

> - tz-get-sanitized command activied, IOC data extracted from report.
> - tz-get-sanitized command returns sanitized file as zip.
> - IOC datas extracted and created a new field as ThreatZone.IOC, suitable for !extractIndicators.

Reviewer: ```suggestion
- Updated the ***tz-get-sanitized*** command to return sanitized file as zip.
```
